README.md

```diff
@@ -20,16 +18,64 @@
+
+Let's go over the parts of the app that are already present in that project.
+
+Open up `Main.storyboard`. You'll notice that a standard view controller has already been created for you. Notice, too, that the light bulb view has also been placed. However, it has _not_ been connected to anything yet—it's up to you to hook it up to the view controller's code properly.
+
+Now open up `ViewController.swift` (you can open it up in an Assistant Editor from Interface Builder if you want). Parts of this file have already been filled in for you, too. It already has an IB action named `colorSelected(_:)`, but that IB action is not hooked up to anything. It also has a `UIView` outlet called `lightBulb`, but that has not been hooked up to anything, either.
+
+You have also been provided with a method for turning strings into colors. For example, you can call `"red".color` to get a red color. `"blue".color`, `"yellow".color`, and `"green".color` are also allowed. This should help you easily create `UIColor` objects and use them in the existing methods.
+
+You can also call `color` on segmented control objects. For example, given a segmented control called `sender` (such as the one passed to `colorSelected(_:`), you can call `sender.color` to get the color associated with it.
+
+Finally, there are three method stubs that have `TODO` markers indicating code that _you_ have to write: `viewDidLoad()`, `changeColor(to:)`, and `colorSelected(_:)`.
+
+Now, on to your tasks.
+
+### Your Tasks
 
-*  Considering this is the first lab encompassing what it is they student learned up to this point in this unit, I think we can make this one slightly easier considering there are scheduled to be two more labs after this one.  
-* Maybe it's something like this [lab](https://github.com/learn-co-curriculum/swift-viewLifeCycle-lab) but not as difficult.  
-* Free to explore ideas or fun projects the student can take on at this point.
+If you complete the following steps, you should end up with a properly working app:
 
+1. Hook up the `lightBulb` IB outlet in `ViewController.swift` with the light bulb view in `Main.storyboard`.
+2. Add a segmented control underneath the lightbulb.
+	* By default, a segmented control only has 2 buttons. Yours needs to have 4. Do you know how you can change that? (If you guessed "in the Attributes Inspector", you'd be correct!)
+	* The individual parts of a segmented control also have default names. Try to figure out on your own how you can change those names. (Hint: Look for a section in the Attributes Inspector called **Segment**. It is a popup menu that should list all the parts of your segmented control. You'll want to change an attribute called **Title** for each segment part.)
+3. Hook up the segmented control to the `colorSelected(_:)` IB outlet in `ViewController.swift`.
```

Comment:
Fixed!

I read through all the comments, so you can keep doing it the way you are. If I happen to miss one that I need to take action on, though, feel free to nudge me.